### DISCUSSION TITLE
BCC folder & runtime folder

### DISCUSSION THREAD

**Discussion by pauloswear** [dashingsoft/pyarmor#1271] [Q&A]
Hello! I have 2 questions:

1º When I obfuscate script with BCC mode enabled, it creates a folder called .pyarmor and inside, bcc folder, that's empty. This folder is empty because i changed output script to desktop:

`pyarmor gen launcher.py --recursive --enable-bcc --mix-str --assert-call --assert-import --output %USERPROFILE%\Desktop\launcher`

Is it normal or it will be improved in next updates? I am waiting pyarmor-8 be stable to change my production.

2º It's possible to not send vax code in runtime folder? When i generate script with old pyarmor-7, it creates a folder called pyarmor, not pyarmor_runtime_vaxcode.

**Comment by jondy** [comment]
You need not care about `.pyarmor` folder, and please check documentation about how to rename runtime package name.

**Reply by VeNoMouS** [reply]
> You need not care about `.pyarmor` folder, and please check documentation about how to rename runtime package name.

I to have this issue, i do not like how it creates a sub folder , i prefer 7.x output

Can you point us in the direction of what youre refering to in the documentation?

**Reply by jondy** [reply]
https://pyarmor.readthedocs.io/en/stable/reference/environments.html#configuration-options

**Reply by jondy** [reply]
And you can change local configuration path 
https://pyarmor.readthedocs.io/en/stable/reference/man.html#cmdoption-pyarmor-home

**Reply by VeNoMouS** [reply]
Thanks, will have a read

**Reply by VeNoMouS** [reply]
mmm sorry, I was refering to the runtime directory that it creates, why cant the files go directly into the dist folder like 7.x instead of a whole new sub folder?

**Reply by jondy** [reply]
@VeNoMouS it could support multiple platforms and multiple Python versions, and could store outer runtime key or other data files in runtime package path.

**Reply by pauloswear** [reply]
@jondy bcc really support multiple platforms?

```
INFO     Python 3.10.5
INFO     Pyarmor 8.2.1 (pro), **, **
INFO     Platform windows.x86_64
INFO     use runtime platforms: ['windows.x86_64', 'windows.x86']
ERROR    bcc mode does not support cross platform
```

**Reply by pauloswear** [reply]
Also, Is there anyway to remove from __init__.py generated file this commented line?
`# Pyarmor 8.2.1 (pro), sufix, 2023-05-26T01:25:51.139973`

**Comment by jondy** [comment]
@pauloswear please check ReleasePlan.md in the project home for cross-platform bcc mode

**Reply by pauloswear** [reply]
Ok, thanks! I've checked.
And about commented line?
# Pyarmor 8.2.1 (pro), sufix, 2023-05-26T01:25:51.139973
